src/interface/inbound/api_template/paths/accounts_ID.yaml

```diff
@@ -0,0 +1,80 @@
+parameters:
```

Comment:
maybe this inbound path and corresponding error are good candidates to be moved to https://github.com/mojaloop/api-snippets/tree/master/thirdparty/openapi3/paths ?

Reply:
I have defined all in local paths, may be we can move all those to `api-snippets` like `sdk-scheme-adapter` in a separate ticket.

Reply:
Indeed. Lets leave it for now. We can refactor paths once complete thirdparty API definitions#2089
is done.